src/memberdef.cpp

```diff
@@ -3033,8 +3037,10 @@ void MemberDefImpl::_writeReimplementedBy(OutputList &ol) const
         {
           //ol.writeObjectLink(bcd->getReference(),bcd->getOutputFileBase(),
           //    bmd->anchor(),bcd->name());
+          SrcLangExt lang = bmd->getLanguage();
+          QCString sep = getLanguageSpecificSeparator(lang,TRUE);
           ol.writeObjectLink(bmd->getReference(),bmd->getOutputFileBase(),
-              bmd->anchor(),bcd->displayName());
+              bmd->anchor(),bcd->displayName() + sep + bmd->name());
 
           if (bmd->isLinkableInProject() )
           {
```

Comment:
I'd not implement this change. The 1.8.17 output ("Implemented in <class>" or "Reimplemented in <class>") makes perfect sense.

Reply:
I added this here so all would have the same output.